fpmsyncd/routesync.h

```diff
@@ -70,6 +75,10 @@ class RouteSync : public NetMsg
                         string& nexthops, string& vni_list, string& mac_list,
                         string& intf_list);
 
+    /* Get next hop list */
+    bool getNextHopList(struct rtnl_route *route_obj, string& gw_list,
+                        string& mpls_list, string& intf_list);
```

Comment:
can you merge getNextHopList with getNextHopGw to avoid code duplication.

Reply:
getNextHopGw() and getNextHopIf() are used by onVnetRouteMsg().  I am not going to change that code.